web/src/components/projects/ProjectCardsList.tsx

```diff
@@ -7,6 +7,7 @@ import styles from "./ProjectCardsList.module.css";
 
 type Props = {
   projects: Project[];
+  isAdminView: boolean;
```

Comment:
I think elsewhere we've (I've) used `isOpsStaff` for this(?) don't mind which it is, so long as it's consistent